cmd/util/defaults.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"time"
+)
+
+const (
+	DefaultLeaderElect                 = true
+	DefaultLeaderElectionNamespace     = "kube-system"
+	DefaultLeaderElectionLeaseDuration = 60 * time.Second
+	DefaultLeaderElectionRenewDeadline = 40 * time.Second
+	DefaultLeaderElectionRetryPeriod   = 15 * time.Second
+)
```

Comment:
You'll need to tell Bazel about this file, and the simplest way is to run `./hack/update-all.sh`.